chef/cookbooks/cinder/templates/default/cinder.conf.erb

```diff
@@ -1644,7 +1644,7 @@ nfs_shares_config=/etc/cinder/nfs_shares-<%= backend_id %>
 
 # Mount options passed to the nfs client. See section of the nfs man page for
 # details. (string value)
-#nfs_mount_options = <None>
+nfs_mount_options = <%= volume['netapp']['nfs_mount_options'] %>
```

Comment:
I think I will add something like this, to avoid change of file:

```ruby
# Mount options passed to the nfs client. See section of the nfs man page for
# details. (string value)
#nfs_mount_options = <None>
<% if volume['netapp']['nfs_mount_options'] -%>
nfs_mount_options = <%= volume['netapp']['nfs_mount_options'] %>
<% end -%>
```